client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1768,7 +1768,7 @@
                     if (angular.isDefined(item)) {
                         item._datelinedate = '';
 
-                        if (angular.isDefined(item.dateline.located) && !_.isNull(item.dateline.located)) {
+                        if (item.dateline && item.dateline.located != null) {
```

Comment:
exactly our linter only allows `!=` and `==` when comparing to `null`, which makes it work with `undefined` as well. otherwise it must be `===`